base/reflection.jl

```diff
@@ -842,9 +849,9 @@ end
 """
     which(symbol)
 
-Return the module in which the binding for the variable referenced by `symbol` was created.
+Return the module in which the binding for the variable referenced by Main.`symbol` was created.
```

Comment:
also a behavior change for calls to this from non-main modules that should be noted more prominently somewhere

and this should maybe be formatted with Main inside the code highlighting - not sure if `Main.$symbol` is quite the right way to show it?